shell/app/common/components/tree/tree.tsx

```diff
@@ -22,6 +22,8 @@ import { EditCategory } from './edit-category';
 import { findTargetNode, getIcon, isAncestor, walkTree } from './utils';
 import { WithAuth } from 'user/common';
 
+import './file-tree.scss';
```

Comment:
file-tree is a module not related with tree. Why do u import this scss file

Reply:
Although the file is named file-tree.scss, it is in the tree folder, and no other file references the file-tree.scss. Should I change its name to tree.scss?